queue_services/entity-filer/src/entity_filer/filing_processors/dissolution.py

```diff
@@ -28,6 +29,16 @@
 from entity_filer.utils import replace_file_with_certified_copy
 
 
+class DissolutionTypes(str, Enum):
+    """Dissolution types."""
+
+    ADMINISTRATIVE = 'administrative'
+    COURT_ORDERED_LIQUIDATION = 'courtOrderedLiquidation'
+    INVOLUNTARY = 'involuntary'
+    VOLUNTARY = 'voluntary'
+    VOLUNTARY_LIQUIDATION = 'voluntaryLiquidation'
```

Comment:
instead of adding this in the filer, can we reference the dissolutionType in legal api?  

https://github.com/bcgov/lear/blob/70528f5b78de3a1be991821f05ed7dbf1380ec4d/legal-api/src/legal_api/services/filings/validations/dissolution.py#L35-L42